gc/gc-tool/src/test/java/org/projectnessie/gc/tool/TestCLI.java

```diff
@@ -64,6 +66,7 @@
 @NessieDbAdapterName(InmemoryDatabaseAdapterFactory.NAME)
 @NessieExternalDatabase(InmemoryTestConnectionProviderSource.class)
 @TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+@NessieApiVersions(versions = NessieApiVersion.V1)
```

Comment:
Can you open a follow-up to let GC use V2?

Reply:
#5422